pkg/shell/nav.scss

```diff
@@ -57,7 +57,7 @@ $desktop: $phone + 1px;
     padding-block: var(--pf-v5-global--spacer--md) var(--pf-v5-global--spacer--sm);
     padding-inline: var(--pf-v5-global--spacer--md);
 
-    > .pf-v5-c-text-input-group__main {
+    >.pf-v5-c-text-input-group__main {
```

Comment:
Please revert these changes (also the ones below), they are unrelated to the caret change and just noise. In general, unrelated changes should be in separate commits.